src/Extensions/HeaderButtonExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace SilverStripe\StartupTheme;
+
+use SilverStripe\Forms\FieldList;
+use SilverStripe\LinkField\Form\LinkField;
+use SilverStripe\LinkField\Models\Link;
+use SilverStripe\ORM\DataExtension;
+
+class HeaderButtonExtension extends DataExtension {
+
+    private static array $has_one = [
+        'HeaderLink' => Link::class
```

Comment:
Please change has_one relation name to HeaderButton to be in line with the class name